imports.go

```diff
@@ -0,0 +1,46 @@
+package cayley
+
+import (
+	"github.com/google/cayley/graph"
+	_ "github.com/google/cayley/graph/memstore"
+	"github.com/google/cayley/graph/path"
+	"github.com/google/cayley/quad"
+	_ "github.com/google/cayley/writer"
+)
+
+type Iterator graph.Iterator
+type QuadStore graph.QuadStore
+type QuadWriter graph.QuadWriter
+
+type Path path.Path
+
+var StartMorphism = path.StartMorphism
+var StartPath = path.StartPath
+
+var RawNext = graph.Next
+
+type Handle struct {
+	graph.QuadStore
+	graph.QuadWriter
+}
+
+func Quad(subject, predicate, object, label string) quad.Quad {
+	return quad.Quad{subject, predicate, object, label}
+}
+
+func NewMemoryGraph() (*Handle, error) {
+	qs, err := graph.NewQuadStore("memstore", "", nil)
+	if err != nil {
+		return nil, err
+	}
+	qw, err := graph.NewQuadWriter("single", qs, nil)
+	if err != nil {
```

Comment:
ok, but *if* it did something then i assume you would need to close it, right?

e.g. does  https://github.com/google/cayley/blob/680bb244d2c7e28dcbf9abe3229f2954dc030130/db/db.go#L45 need to `qs.Close()`?